HACKING.md

```diff
@@ -102,9 +102,19 @@ Install playwright and its browsers
 
     npx playwright install
 
-The tests expect the environment on localhost to be accessible. Execute `dotrun` and start tests with
+The e2e tests can be run against the LTS version (v5.0) or the latest release version for LXD. If you want to run the tests against the edge version, first make sure your lxd is up to date with
```

Comment:
```suggestion
The e2e tests can be run against the LXD 5.0, or the edge version of LXD. If you want to run the tests against the edge version, first make sure your lxd is up to date with
```
Maybe better to not assume 5.0 is "the LTS", as it will be a new LTS soon.